samples/word/07-custom-properties/custom-properties-access.yaml

```diff
@@ -1,24 +1,26 @@
-id: word-basics-custom-properties-access
+id: word-basics-custom-properties-access-
 name: Get Custom Document Properties
 description: Shows how to retrieve the current document's custom properties.
 author: Microsoft
 host: WORD
 api_set:
     WordApi: 1.1
 script:
-    content: |4
-          $("#run").click(getProperties);
+    content: |
+        $("#run").click(getProperties);
 
-            function getProperties() { 
-                Word.run(function (context) {
-                    var customDocumentProperties = context.document.properties.customProperties;
-                    context.load(customDocumentProperties);
-                    return context.sync()
-                        .then(function () {
-                            console.log(customDocumentProperties.items.length);
-                         })
-                 })
-            }
+        function getProperties() {
+            Word.run(function (context) {
+                var customDocumentProperties = context.document.properties.customProperties;
+                context.load(customDocumentProperties);
```

Comment:
Here an elsewhere, not specifying the property names you want is BAD PRACTICE.  More about it in Chapter 5 of my book, which you should have access to (if not, email me, and I'll send you a link for a free copy for our team).